Feature wishlist permissions

S1onyx opened the pull request.

S1onyx committed:
getWishlistPermissions

S1onyx committed:
Add the postWishlistPermission endpoint

S1onyx committed:
add put and delete permission endpoint

MrChemical1337 commented on test/bruno/wishlists/permissions/putWishlistPermission.bru:
add json body for testing

MrChemical1337 commented on backend/models/wishlists/permissions/postPermissionModel.js:
owner shouldnt be able to give himsld red / write rights

S1onyx committed:
fix the requested features
owner cannot add his own permissions in the permission table. The owner is always the owner!

MrChemical1337 reviewed (dismissed):
Looks good (works)

S1onyx committed:
Merge branch 'dev' into feature-wishlistPermissions
